Makefile

```diff
@@ -35,24 +35,35 @@ help:
 list:
 	@echo ${CITIES}
 
-%.osm.pbf:
-	mkdir -p ${DATA_DIR}
-	echo "Downloading $(notdir $*) from BBBike."
-	wget -U headway/1.0 -O $@ "https://download.bbbike.org/osm/bbbike/$(notdir $*)/$(notdir $@)" || rm $@
-	@echo "\n\nConsider donating to BBBike to help cover hosting! https://extract.bbbike.org/community.html\n\n"
+$(filter %,$(CITIES)): %: \
+		${DATA_DIR}/%.bbox \
+		${DATA_DIR}/%.osm.pbf \
+		${DATA_DIR}/%.gtfs.tar \
+		${DATA_DIR}/%.nominatim.sql ${DATA_DIR}/%.nominatim_tokenizer.tgz \
```

Comment:
I left them together since they are created together; the inconsistency was intentional. https://github.com/ellenhp/headway/pull/38#discussion_r886776958

Reply:
Happy to do it either way.